python/cuspatial/cuspatial/core/spatial_window.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+from cudf import DataFrame
+from cudf.core.column import as_column
+
+from cuspatial._lib import spatial_window
+
+
+def points_in_spatial_window(min_x, max_x, min_y, max_y, xs, ys):
+    """ Return only the subset of coordinates that fall within the numerically
+    closed borders [,] of the defined bounding box.
```

Comment:
The implementation indicates that the borders would be open, I think? (I'm still fuzzy about open/closed set terminology).
`return x > min_x && x < max_x && y > min_y && y < max_y;`

It's possible we may want to use `>=`/`<=` in the implementation?

Reply:
I actually didn't understand what that meant in the docs (`[, ]` is not obvious!). Now I do.  

I don't think "closed" is a useful term here. Typically "inclusive/inclusive" are used. Which does that mean? The implementation always used < not <=.   <= might be more intuitive...  But I'm worried we're going to get the usual misunderstandings about floating point representation...

Reply:
I fixed the documentation to match the real behavior (<, not <=) and not use vague terms.

Reply:
My understanding is that a closed interval `[a, b]` is a closed set.

(I don't think the inverse is true always).